autoscale/src/main/java/com/sequenceiq/periscope/monitor/evaluator/flow/FlowEvaluator.java

```diff
@@ -0,0 +1,89 @@
+package com.sequenceiq.periscope.monitor.evaluator.flow;
+
+import java.time.Instant;
+import java.util.List;
+
+import javax.annotation.Nonnull;
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.periscope.domain.Cluster;
+import com.sequenceiq.periscope.monitor.context.ClusterIdEvaluatorContext;
+import com.sequenceiq.periscope.monitor.context.EvaluatorContext;
+import com.sequenceiq.periscope.monitor.evaluator.EvaluatorExecutor;
+import com.sequenceiq.periscope.monitor.evaluator.EventPublisher;
+import com.sequenceiq.periscope.monitor.event.ScalingTriggerCleanupEvent;
+import com.sequenceiq.periscope.monitor.event.ScalingTriggerFlowStatusEvent;
+import com.sequenceiq.periscope.service.ClusterService;
+import com.sequenceiq.periscope.service.ScalingTriggerService;
+
+@Component("FlowEvaluator")
+@Scope("prototype")
+public class FlowEvaluator extends EvaluatorExecutor {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(FlowEvaluator.class);
+
+    private static final String EVALUATOR_NAME = FlowEvaluator.class.getName();
+
+    private long clusterId;
+
+    @Value("${periscope.scalingTrigger.cleanup-duration.minutes}")
```

Comment:
A default cleanup interval of 60 minutes seems extremely low. Likely at least a day. Is there a reason to go with 60 minutes ?

Reply:
I added this as a sample value to test out the cleanup implementation. Changed it to a day in the corresponding patch